workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Readers/IReaderInternal.cs

```diff
@@ -1,6 +1,9 @@
-﻿using Improbable.Worker;
+﻿using System.Runtime.CompilerServices;
+using Improbable.Worker;
 
-namespace Improbable.Gdk.Core.MonoBehaviours
+[assembly: InternalsVisibleTo("Improbable.Gdk.Core.EditmodeTests")]
```

Comment:
Move this into its own `AssemblyInfo.cs`, since it's global to the entire assembly.